frontend/public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="https://cdn.prod.website-files.com/671156d33ac264346e223043/671af30cbd385810328a2bea_favi%20(19).png" />
```

Comment:
Please use the locally hosted version not a separate CDN, we will get caching via cloudflare given etag cache expiry

Reply:
Okay, sure. I will do it.